### DISCUSSION TITLE
Handling a click on a disabled button

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3301] [Q&A]
### Question

I was confused with the fact that I need to process a click on a switched off button, or rather a long press

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
I can't understand what you mean.
Could you be more specific?

**Reply by RRozi** [reply]
lol, what’s not clear? Need to handle disabled button

**Reply by ndonkoHenri** [reply]
when a button is disabled, `on_click` is not called

**Reply by RRozi** [reply]
He discovered America... This is the essence of the question. I needed to solve this so that I could use similar functionality despite the button being disabled
